KPP/fullchem/fullchem_RateLawFuncs.F90

```diff
@@ -1822,7 +1822,8 @@ FUNCTION HOBrUptkByBrSALA( H ) RESULT( k )
        ENDIF
        !
        ! Uptake rate [1/s]
-       area = H%ClearFr * H%aClArea
+!       area = H%ClearFr * H%aClArea
+       area = H%ClearFr * H%aClArea * (1.0_dp - H%f_Alk_SSA)
```

Comment:
It might be worth adding new fields to State_Het to compute the 1.0 - f_alk_SSA and 1.0 - f_Alk_SSC.  That would avoid doing these computations on every (I,J,L) iteration.

Reply:
The new fields would be set in routine fullchem_SetStateHet (in fullchem_HetStateFuncs.F90).  If you like I could implement them and push to the branch.

Reply:
This is now fixed in 23d02914fd787733b6103fe81a34ddd35606bd58